contracts/modules/royalty/policies/IpRoyaltyVault.sol

```diff
@@ -207,7 +207,7 @@ contract IpRoyaltyVault is IIpRoyaltyVault, ERC20Upgradeable, ReentrancyGuardUpg
     /// @param claimer The address of the claimer
     /// @param token The revenue token to check
     /// @return The revenue debt of the claimer
-    function claimerRevenueDebt(address claimer, address token) external view returns (int256) {
+    function revenueDebt(address claimer, address token) external view returns (int256) {
```

Comment:
can we have the same name for variable and for view function name for consistency?

right now `revenueDebt` in view function and `claimerRevenueDebt` in variable. 

Could be either one as long as they are the same for consistency